README.md

```diff
@@ -184,6 +184,30 @@ The main configuration file for the operation of ORT. This configuration is main
 the ORT instance. In contrast to the configuration files in the following, this file rarely changes once ORT is
 operational.
 
+The reference configuration file linked via the header of this section gives a good impression about the content of
```

Comment:
Commit message *title*: Please use

    README: Improve documentation of the ORT config file